letsencrypt-win-simple/Web_Config.xml

```diff
@@ -5,9 +5,9 @@
 			<mimeMap fileExtension="*" mimeType="text/json" />
 		</staticContent>
 	</system.webServer>
-  <system.web>
```

Comment:
You need to undo your changes to the web_config. They are preventing an automatic merge, since the web_config has been changed.